riemann/roles/network/files/etc/network/interfaces.d/riemann

```diff
@@ -0,0 +1,15 @@
+# Managed by Ansible
```

Comment:
I've switched to using `{{ ansible_managed }}` instead of hand-coding this. Not sure why.

Reply:
rettet!